pokemongo_bot/api_wrapper.py

```diff
@@ -61,13 +61,16 @@ def call(self, max_retry=5):
             self._api._req_method_list = [req_method for req_method in api_req_method_list] # api internally clear this field after a call
             result = self._api.call()
             if not self._is_response_valid(result, request_callers):
-                try_cnt += 1
-                logger.log('Server seems to be busy or offline - try again - {}/{}'.format(try_cnt, max_retry), 'red')
-                if try_cnt >= max_retry:
-                    raise ServerBusyOrOfflineException()
-                sleep(1)
-            else:
-                break
+                while try_cnt < max_retry:
+                    try_cnt += 1
```

Comment:
Seems that way to me as well.  The reason why we are seeing 1/5 so many times is because each one of those is a separate call, not because the counter is broken:

```java
[17:23:16]
called: ['GET_MAP_OBJECTS']
called: ['GET_GYM_DETAILS']
called: ['GET_GYM_DETAILS']
called: ['GET_GYM_DETAILS']
[17:23:16] Server seems to be busy or offline - try again - 1/5
called: ['GET_GYM_DETAILS']
called: ['GET_INVENTORY', 'GET_PLAYER']
[17:23:18] Starting PokemonGo Bot....
called: ['GET_MAP_OBJECTS']
called: ['GET_PLAYER']
called: ['GET_INVENTORY']
[17:23:18] Server seems to be busy or offline - try again - 1/5
[17:23:19] [x] Found fort 3766dd707bed47a389d3654a247ad65d.16 at distance 75.46ft
[17:23:19] [x] Need to move closer to Pokestop
called: ['GET_PLAYER', 'GET_HATCHED_EGGS', 'CHECK_AWARDED_BADGES']
called: ['GET_MAP_OBJECTS']
called: ['GET_GYM_DETAILS']
called: ['GET_GYM_DETAILS']
called: ['GET_GYM_DETAILS']
[17:23:20] Server seems to be busy or offline - try again - 1/5
called: ['GET_GYM_DETAILS']
called: ['GET_MAP_OBJECTS']
called: ['GET_PLAYER']
called: ['GET_INVENTORY']
[17:23:23] [x] Found fort 3766dd707bed47a389d3654a247ad65d.16 at distance 65.02ft
[17:23:23] [x] Need to move closer to Pokestop
called: ['GET_PLAYER', 'GET_HATCHED_EGGS', 'CHECK_AWARDED_BADGES']
called: ['GET_MAP_OBJECTS']
called: ['GET_GYM_DETAILS']
called: ['GET_GYM_DETAILS']
[17:23:24] Server seems to be busy or offline - try again - 1/5
called: ['GET_GYM_DETAILS']
called: ['GET_GYM_DETAILS']
called: ['GET_MAP_OBJECTS']
called: ['GET_PLAYER']
called: ['GET_INVENTORY']
```